iot/api-client/manager/manager.py

```diff
@@ -216,10 +216,10 @@ def delete_registry(
     try:
         response = client.delete_device_registry(registry_path)
```

Comment:
```suggestion
        client.delete_device_registry(registry_path)
```

Lint is complaining about the unused variable.

```
Running session lint
Creating virtual environment (virtualenv) using python3.6 in .nox/lint
pip install flake8 flake8-import-order
flake8 --show-source --builtin gettext --max-complexity 20 --import-order-style google --exclude .nox,.cache,env,lib,generated_pb2,*_pb2.py,*_pb2_grpc.py --ignore=E121,E123,E126,E203, E226,E24,E266,E501,E704,W503,W504,I100,I201,I202 --application-import-names manager_test,noxfile,resources,manager,.nox .
./manager.py:217:9: F841 local variable 'response' is assigned to but never used
        response = client.delete_device_registry(registry_path)
        ^
Command flake8 --show-source --builtin gettext --max-complexity 20 --import-order-style google --exclude .nox,.cache,env,lib,generated_pb2,*_pb2.py,*_pb2_grpc.py --ignore=E121,E123,E126,E203, E226,E24,E266,E501,E704,W503,W504,I100,I201,I202 --application-import-names manager_test,noxfile,resources,manager,.nox . failed with exit code 1
Session lint failed.

 Testing failed: Nox returned a non-zero exit code.

cleanup
```

Reply:
![yikes](https://media.tenor.com/images/b577fac963e22f691f883fe537490a58/tenor.gif)

I should have caught that one on initial PR, thanks for reviewing.